e2e/next/src/next.test.ts

```diff
@@ -463,6 +463,17 @@ describe('Next.js Applications', () => {
       checkExport: false,
     });
   }, 300_000);
+
+  it('should support --turbo to enable Turbopack', async () => {
+    const appName = uniq('app');
+
+    runCLI(
+      `generate @nx/next:app ${appName} --style=css --appDir --no-interactive`
+    );
+
+    const result = runCLI(`serve ${appName}`);
+    expect(result).toContain('>>> TURBOPACK (beta)');
```

Comment:
Maybe remove the `(beta)` part so when it is stable it would still work? I guess since we're checking strings it could change at any point anyway.